### ISSUE TITLE
Not sure what I'm doing wrong =(

### ISSUE THREAD

**Issue by DarkFalcon81** [long-war-2/lwotc#691]
Trying my hardest to get LWOTC installed but I just can't make it happen it seems.... yes, I have read the instructions, yes, i have created the Mods folder and put the files in the right place, but, it doesn't matter.. the mod manager just doesn't recognize the mod when i start up the game.... any ideas?

**Comment by ansabalirai** [comment]
If you are using alterative mod launcher, have you made sure to add the relevant directories in settings so it knows where to look for?

**Comment by DarkFalcon81** [comment]
Thank you for your reply ansa. But nope, I'm not using the alternative mod launcher... should I?

**Comment by DarkFalcon81** [comment]
Ok.. I just tried installing the alternative launcher, I added the directory to where LWOTC and its highlander mod is in the settings... yet nothing... doesn't show up there either... all other mods show...

**Comment by DarkFalcon81** [comment]
Yeah it's absolutely not showing up in either launcher. Checked and double checked so many times now. Don't know what else to do.

**Comment by Mejker10** [comment]
If you're using discord i might be able to walk you through the process to make sure you're not doing something wrong/missing a step. Add Mejker#7300.

**Comment by pledbrook** [comment]
Try the instructions here: https://docs.google.com/document/d/1BkdQn37MXsJP81v49z_x2vyhqVt0AZiBrcyCWr2EsQE/edit

Also, please provide the full paths to the _LongWarOfTheChosen_ and _X2WOTCCommunityHighlander_ directories (which you should have unpacked from the lwotc zip).

**Comment by DarkFalcon81** [comment]
Thank you so much guys for the help! I've got it working now perfectly! My apologies, I was just being a complete dum-dum... what was causing this issue? Well... not downloading the right zip file... ;) The zip file I had downloaded and put in the mods folder was the lwotc-content.zip not the LongWaroftheChosen-1.0-beta.zip which is obviously the right one. So, no wonder I couldn't get it to work. Well, I learned my lesson there, that was the last time I downloaded and tried to install something while sleep-deprived. =)
What is the content thing tho? Dev/modder only stuff I suppose?
